docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -107,40 +149,48 @@ Both offer monitoring of a single process instance, its variables and path, inci
 
 ## Limitations and availability
 
-:::note
-Play is being rebuilt and progressively rolled out to more users. This section explains why you might not see the **Play** tab.
-:::
+Play is being rebuilt and progressively rolled out to more users. This section explains why you might not see the **Play** tab, and any additional limitations.
 
-For Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
-Enterprise users need an admin to enable Play by opting in to [alpha features](/components/console/manage-organization/enable-alpha-features.md).
+For more information about terms, refer to our [licensing and terms page](https://legal.camunda.com/licensing-and-other-legal-terms#c8-saas-trial-edition-and-free-tier-edition-terms).
+
+### Camunda 8 SaaS
 
-For Self-Managed, Play is controlled by the **PLAY_ENABLED** flag. It is `true` by default for the Docker distribution for development and `false` by default on the Kubernetes distribution for production use.
+In Camunda 8 SaaS, Play is available to all Web Modeler users with editor or admin permissions within a project.
+Enterprise users need an admin to enable Play by opting in to [alpha features](/components/console/manage-organization/enable-alpha-features.md).
```

Comment:
This should be removed with 8.6.0

All users will have access to Play, but should be able to toggle (for a limited period) whether new Play or old Play is shown